modules/nf-core/atlasgeneannotationmanipulation/gtf2featureannotation/main.nf

```diff
@@ -0,0 +1,41 @@
+process ATLASGENEANNOTATIONMANIPULATION_GTF2FEATUREANNOTATION {
+    tag '$gtf'
+    label 'process_single'
+
+    conda (params.enable_conda ? "bioconda::atlas-gene-annotation-manipulation=1.1.0" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/atlas-gene-annotation-manipulation%3A1.1.0--hdfd78af_0':
+        'quay.io/biocontainers/atlas-gene-annotation-manipulation:1.1.0--hdfd78af_0' }"
+
+    input:
+    path gtf
```

Comment:
I think you should add a meta map here because process inputs in nf-core pipelines typically carry sample specific information in them. 
https://nf-co.re/docs/contributing/modules#what-is-the-meta-map

Reply:
This is a gtf I imagined being kind of global to a workflow, which is why I didn't do that.

But maybe there could be multiple species/ annotation types. I'll add the meta.